### PR TITLE
Fix CMake generator expression syntax

### PR DESCRIPTION
**Pull request by byrnHDF**

### CODE REVIEW

**Review by bmribler — APPROVED** [review]

**Review by bmribler — APPROVED** [review]

File: CMakePresets.json [hunk 1]
```diff
@@ -182,7 +182,12 @@
       "configurePreset": "ci-StdShar-MSVC",
       "inherits": [
         "ci-x64-Release-MSVC"
-      ]
+      ],
+      "filter": {
+        "exclude": {
+          "name": "H5DUMP-tfloatsattrs"
```
Comment (jhendersonHDF) [inline CMakePresets.json:188]: As an aside, is this test excluded due to floating point error differences between testing on Linux and Windows or due to some other reason?

**Review by jhendersonHDF — APPROVED** [review]

File: CMakePresets.json (hunk 1 above)
Comment (byrnHDF) [reply CMakePresets.json:188]: Correct.

Comment (jhendersonHDF) [reply CMakePresets.json:188]: Interesting - the H5Z ZFP tests run into the same issue with floating point accuracy on Windows. Would need to do some research to figure out if the results can be made close enough to identical across platforms.

Comment (byrnHDF) [reply CMakePresets.json:188]: This test just runs into rounding errors - ZFP I think has bigger differences

**Pull request merged**